packages/venia-concept/package.json

```diff
@@ -144,6 +146,9 @@
   "pwa-studio": {
     "targets": {
       "intercept": "./local-intercept.js"
-    }
+    },
+    "trusted-vendors": [
+      "@gomage"
```

Comment:
Shouldn't this get pulled into @magento instead? Alternatively anything published as gomage https://www.npmjs.com/search?q=gomage would come into scope.